### PR TITLE
Update Traditional Chinese edition to align with latest English CfP text

### PR DESCRIPTION
**Pull request by CYBAI**
The English update is at https://github.com/rustlang-tokyo/RustFestGlobal2020-CFP/compare/master...flaki:master.

Feel free to let me know if there's any wording can be better!!

### CODE REVIEW

File: zh_tw.md [hunk 1]
```diff
@@ -82,26 +81,32 @@ RustFest 非常重視易參與性。再者，因為此次為虛擬研討會，
 
 如果您被選為 RustFest 講者，您將得到：
 
-- 參加虛擬研討會的權限。如果您被選為講者且您已購票，我們可以進行退款，但如果您覺得不需要退款，我們將有更多的額度來使這個研討會更棒更便利 <3。
-- 買齊必須裝備來讓演講更棒。例如：如果您需要一個視訊鏡頭或是麥克風，我們可以一起討論如何協助您得到一個。
+- 如果您被選為講者且您已購票，我們可以進行退款，但如果您覺得不需要退款，我們將有更多的額度來使這個研討會更棒更便利 <3。
+-
```
Comment (kanru) [inline zh_tw.md]: Extra item.

File: zh_tw.md [hunk 2]
```diff
@@ -82,26 +81,32 @@ RustFest 非常重視易參與性。再者，因為此次為虛擬研討會，
 
 如果您被選為 RustFest 講者，您將得到：
 
-- 參加虛擬研討會的權限。如果您被選為講者且您已購票，我們可以進行退款，但如果您覺得不需要退款，我們將有更多的額度來使這個研討會更棒更便利 <3。
-- 買齊必須裝備來讓演講更棒。例如：如果您需要一個視訊鏡頭或是麥克風，我們可以一起討論如何協助您得到一個。
+- 如果您被選為講者且您已購票，我們可以進行退款，但如果您覺得不需要退款，我們將有更多的額度來使這個研討會更棒更便利 <3。
+- 
+
+- 參加虛擬研討會的完整權限。如果您已購票，我們可以提供退款服務。
```
Comment (kanru) [inline zh_tw.md]: s/權限/體驗/

File: zh_tw.md [hunk 3]
```diff
@@ -82,26 +81,32 @@ RustFest 非常重視易參與性。再者，因為此次為虛擬研討會，
...
+>
+> 這場議程將重新檢視相關的原則並一探真實世界的範例，當你聽完這場議程，將會有顆「寫文件有助於你的使用者及軟體生存」的實戰經驗之心確認清單。
 
 ## 其他
 
-除了直播之外，所有的議程都將被錄影、紀錄講稿、還有錄影的投影片、live demo 及其他講者螢幕上的互動都將被免費公布至網路上。
+除了直播之外，所有的議程都將被錄影、紀錄講稿、還有錄影的投影片、live demo 及其他講者螢幕上的互動都將（在經過您的允許及確認後）被免費公布至網路上。
+Rust 的 YouTube 頻道是由 [Rust Community team](https://www.rust-lang.org/community) 經營管理。
 
 我們這麼做是希望讓廣大的 Rust 社群都能受益於這些演講，並且希望無法即時參加研討會的朋友也能夠觀看各個議程。我們希望您也能夠接受議程被錄影等事項，但如果有任何讓您感到不適的，請讓我們知道，我們將會想辦法解決。
```
Comment (kanru) [inline zh_tw.md:110]: 之前沒有注意到, Creative Commons 直接翻成 `創用CC (Creative Commons)` 就好了，CC0 只是其中一種。

File: zh_tw.md (hunk 1 above)
Comment (CYBAI) [reply zh_tw.md]: oops 😂

File: zh_tw.md (hunk 3 above)
Comment (CYBAI) [reply zh_tw.md:110]: ohoh! 原來！感謝！之前不知道細節（筆記筆記

**Review by kanru — APPROVED** [review]

**Comment by yytyd** [comment]
Thanks!

**Pull request merged**